stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/DropdownFieldController.kt

```diff
@@ -27,7 +27,7 @@ class DropdownFieldController(
     val selectedIndex: StateFlow<Int> = _selectedIndex
     override val label: StateFlow<Int> = MutableStateFlow(config.label)
     override val fieldValue = selectedIndex.mapAsStateFlow { displayItems[it] }
```

Comment:
theoretically, this could still crash because of the `displayItems[it]` here. If it does, we would get notified via SDK console again